deployment/secure_research_environment/cloud_init/cloud-init-srd.mustache.yaml

```diff
@@ -426,7 +426,7 @@ write_files:
         BLOBFUSE_CACHE_DIR="/tmp/blobfuse-cache-egress"
         rm -rf $BLOBFUSE_CACHE_DIR
         mkdir -p $BLOBFUSE_CACHE_DIR
-        CACHE_SPACE_MB=$(echo "$(df -BM | grep /mnt | awk '{print $2}' | sed 's/M//') / 2" | bc) # set the cache size to half the size of /mnt which scales with VM size
+        CACHE_SPACE_MB=$(echo "$(df -BM | grep /dev/root | awk '{print $2}' | sed 's/M//') / 50" | bc) # set the cache size to 2% of the OS disk size
```

Comment:
```suggestion
        CACHE_SPACE_MB=$(echo "$(findmnt -nb -o size /) / 1024^2 / 50" | bc) # set the cache size to 2% of the OS disk size
```

Reply:
Is `/dev/root` guaranteed to be what we want? I've not come across that before (doesn't seem to exist on any of my machines).

This way we are asking, what is the size of the device mounted at `/` rather than assuming the name of the device in `/dev`.

Reply:
Good idea. Can you confirm that these two give the same results?

Reply:
For me
```
❯ echo "$(findmnt -nb -o size /) / 1024^2 / 50" | bc
4662
❯ echo "$(df -BM | grep /dev/nvme0n1p2 | awk '{print $2}' | sed 's/M//') / 50" | bc
4662
```